charts/kcp/templates/external-logical-cluster-admin-kubeconfig.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+stringData:
+  kubeconfig: |
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority: /etc/kcp-front-proxy/tls/tls.crt
+        server: https://{{ .Values.externalHostname }}:443
+      name: external-logical-cluster:admin
```

Comment:
I'm curious if the `:` in `cluster:admin` is intentional or supposed to be a `-`?